XiEditor/EditViewController.swift

```diff
@@ -32,9 +32,20 @@ class EditViewController: NSViewController, EditViewDataSource {
     @IBOutlet weak var editContainerView: EditContainerView!
     @IBOutlet var editView: EditView!
     @IBOutlet weak var gutterView: GutterView!
+    @IBOutlet weak var stackView: NSStackView!
```

Comment:
I have to play around with the alternative, but I guess the alternative is changing the constraints of the EditView programatically according to whether the FindBar is visible or not? If so, isn't the StackView exactly build for this use case?